packages/central-server/src/apiV2/utilities/meditrakSync/permissionsBasedMeditrakSyncQuery.js

```diff
@@ -32,14 +31,20 @@ export const permissionsFreeChanges = since => {
 };
 
 export const changesWithPermissions = (countryIds, permissionGroups, since) => {
-  let query = 'change_time > ? AND "type" = ?';
-  const params = [since, 'update'];
+  let query = `change_time > ? AND "type" = ? and record_type NOT IN ${SqlQuery.record(
+    recordTypesToAlwaysSync,
+  )}`;
```

Comment:
This `record_type NOT IN recordTypesToAlwaysSync` clause would have been valid previously as well honestly. It's just that with the new changes to always sync when the `countryIds` is null, we keep re-syncing permission groups that we've already synced